lonestar/analytics/cpu/k-core/kcore_cli.cpp

```diff
@@ -79,10 +79,11 @@ main(int argc, char** argv) {
   total_timer.start();
 
   if (!symmetricGraph) {
```

Comment:
I think it is better to show the following warning if the flag is set not vice versa.

Reply:
Yeah makes sense. Good Catch.